notebooks/gpu-footprint.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<p>It seems that the output for that run was somehow not logged here?</p><p> </p><p>Why not just add this model with the others above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The RAM was 8gb at that point and this wasn't able to fit in that. I'll update it now.</p>